lib/composer/monolog/monolog/src/Monolog/Formatter/NormalizerFormatter.php

```diff
@@ -116,6 +119,18 @@ protected function normalizeException(Exception $e)
         return $data;
     }
 
+    protected function normalizeStatus(Status $status)
+    {
+        $data = array(
+            'is_ok' => $status->isOK(),
+            'errors' => $status->getErrorsArray(),
+            'message' => $status->getMessage(),
+            'value' => $status->value,
+        );
+
+        return $data;
+    }
+
```

Comment:
Wrap this block in `Wikia change - begin` / `Wikia change - end` comments and we're good